lib/omniauth/onelogin_setup.rb

```diff
@@ -0,0 +1,41 @@
+module OneloginSetup
```

Comment:
This module would be a great candidate for a DfE gem 👍 following this guide 
https://github.com/omniauth/omniauth/wiki/Strategy-Contribution-Guide